hoodie-utilities/pom.xml

```diff
@@ -31,7 +31,61 @@
   </properties>
 
   <build>
+    <pluginManagement>
```

Comment:
@abhioncbr : Looks like this could be a bad merge. hoodie-utilities is no longer a shaded fat jar. We have packaging/hoodie-utilities-bundle to do that. Can you revert this pom change.